tests/e2e-pw/specs/merchant/merchant-orders-manual-capture.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { getShopper, getMerchant } from '../../utils/helpers';
+import {
+	goToOrder,
+	goToWooPaymentsSettings,
+} from '../../utils/merchant-navigation';
+import { saveWooPaymentsSettings } from '../../utils/merchant';
+import {
+	emptyCart,
+	fillCardDetails,
+	focusPlaceOrderButton,
+	placeOrder,
+	setupProductCheckout,
+} from '../../utils/shopper';
+import { goToShop } from '../../utils/shopper-navigation';
+
+/**
+ * Local variables.
+ */
+let orderId;
+
+test.describe( 'Order > Manual Capture', () => {
+	test.beforeEach( async ( { browser } ) => {
+		// Merchant go to settings, enable capture later, and then save.
+		const { merchantPage } = await getMerchant( browser );
+		await goToWooPaymentsSettings( merchantPage );
+		await merchantPage.getByTestId( 'capture-later-checkbox' ).click();
+		await merchantPage
+			.getByRole( 'button', { name: 'Enable manual capture' } )
+			.click();
+		await saveWooPaymentsSettings( merchantPage );
+
+		// Shopper add items to cart, fill in the checkout, place an order.
+		const { shopperPage } = await getShopper( browser );
+		await emptyCart( shopperPage );
+		await goToShop( shopperPage, 1 );
+		await setupProductCheckout( shopperPage );
+		await fillCardDetails( shopperPage );
+		await focusPlaceOrderButton( shopperPage );
+		await placeOrder( shopperPage );
+
+		// Confirm that the order was placed and get the order number.
+		await shopperPage.waitForURL( /\/order-received\//, {
+			waitUntil: 'load',
+		} );
+		await expect(
+			shopperPage.getByRole( 'heading', { name: 'Order received' } )
+		).toBeVisible();
+		const orderIdField = shopperPage.locator(
+			'.woocommerce-order-overview__order.order > strong'
+		);
+		orderId = await orderIdField.innerText();
+	} );
+
+	test.afterEach( async ( { browser } ) => {
+		// Merchant go to settings, disable capture later, and then save.
+		const { merchantPage } = await getMerchant( browser );
+		await goToWooPaymentsSettings( merchantPage );
+		await merchantPage.getByTestId( 'capture-later-checkbox' ).click();
+		await saveWooPaymentsSettings( merchantPage );
+	} );
+
+	test( 'should create an "On hold" order then capture the charge', async ( {
+		browser,
+	} ) => {
+		// Merchant go to the order.
+		const { merchantPage } = await getMerchant( browser );
+		await goToOrder( merchantPage, orderId );
+
+		// Confirm order status is 'On hold', and that there's an 'authorized' note.
+		await expect( merchantPage.getByTitle( 'On hold' ) ).toHaveText(
+			'On hold'
+		);
+		await expect(
+			merchantPage.getByText(
+				/A payment of \$\d+\.\d{2}.* was authorized using WooPayments/
+			)
+		).toBeVisible();
+
+		// Set select to 'capture_charge', submit, and confirm 'captured' order note.
+		merchantPage
+			.locator( '#woocommerce-order-actions select' )
+			.selectOption( 'capture_charge' );
+		// Using locator due to there are several buttons "named" Update.
+		merchantPage
+			.locator( '#woocommerce-order-actions li#actions button' )
+			.click();
```

Comment:
We need to be very careful, it's easy to forget about adding `await` to every Playwright action and quite frustrating when you can't find why the test is failing 😅

Reply:
Thanks for the call out here. I likely misunderstood something somewhere and removed the `await`. I went back ovee the docs, as I thought there was something about "auto waiting", but I am not sure exactly where I saw that. 

Added the `await` tags in https://github.com/Automattic/woocommerce-payments/pull/10180/commits/22412db0728f4d4a1b443fab238494f237bbdb39

Reply:
Yep, you probably mean [Playwright Auto-waiting docs](https://playwright.dev/docs/actionability). It means that under the hood, it waits for the element to be visible, enabled, etc. so we don't need to manually do it. But these `await` are just the JS syntax to wait for a promise to resolve, avoiding a promise chain.

If you check any Playwright method definition, you'll notice that they always return a promise.